good-enough-practices-for-scientific-computing.tex

```diff
@@ -784,14 +829,14 @@ \subsection*{Acknowledgments}
 file when and by whom, and synchronize changes to a central server so
 that many users can manage changes to the same set of files.
 
-Although all of the authors use version control daily for all of their
-projects, we recognize that many newcomers to computational science
-find version control to be one of the more difficult practices to
-adopt.  We therefore recommend that projects adopt \emph{either} a
-systematic manual approach for managing changes \emph{or} version
-control in its full glory.
+While these version control tools make keeping track of changes easier and
+automatically integrates it into the workflow, they can have a steep learning
+curve. So, we provide two sets of recommendations, \emph{1} a
+systematic manual approach for managing changes and \emph{2} version
```

Comment:
(1) instead of \emph{1} and similarly for 2 - the difference between normal '1' and italics '1' isn't noticeable to me.